<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>InlineReplace</class>
 <widget class="QWidget" name="InlineReplace" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>74</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="6" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>31</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" >
    <widget class="QLineEdit" name="editSearchText" />
   </item>
   <item row="1" column="2" >
    <widget class="QLineEdit" name="editReplaceText" />
   </item>
   <item row="1" column="0" >
    <widget class="QToolButton" name="buttonHide" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="qtedit4.qrc" >:/images/fileclose.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="4" >
    <widget class="QCheckBox" name="cbReplaceAll" >
     <property name="text" >
      <string>&amp;Replace all</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" >
    <widget class="QCheckBox" name="cbMatchCase" >
     <property name="text" >
      <string>&amp;Match case</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QPushButton" name="buttonReplace" >
     <property name="text" >
      <string>Replace</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Replace with:&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Text to replace:&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="0" column="5" >
    <widget class="QCheckBox" name="cmPrompt" >
     <property name="text" >
      <string>&amp;Prompt on replace</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <widget class="QWidget" name="" />
  <widget class="QWidget" name="" />
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="qtedit4.qrc" />
 </resources>
 <connections/>
</ui>
